src/api/lib/tasks/statistics/github/code_frequency.rake

```diff
@@ -0,0 +1,30 @@
+require 'net/http'
+
+namespace :statistics do
+  namespace :github do
+    desc 'Exports the number of additions and deletions per week to a file'
+    task code_frequency: :environment do
+      uri = URI('https://api.github.com/repos/openSUSE/open-build-service/stats/code_frequency')
+      response = Net::HTTP.get_response(uri)
+
+      # The first time you make this request to github, it returns an empty json response with status 202
+      # because github generates the statistics asynchronously. Once they are generated will be 200.
+      if response.code == '202'
+        puts 'Statistics are being generated in the background by github. Please re-run this task in a minute to get the results.'
```

Comment:
Maybe as a little feedback but I'm not against keeping it as it is: How about not exiting the task here but instead sending another HTTP request in 5 seconds so that you don't have to run it manually?

( Would be just the cherry on the ice cream but maybe you find this interessting ;-) )